requirements.md

```diff
@@ -28,9 +28,9 @@ The following Android devices are supported by AGI:
 
 Device name                        | GPU name
 ---------------------------------- | -------------------
-Google Pixel 4 (standard and XL)   | Qualcomm Adreno 640
+Google Pixel 4 (standard and XL)   | Qualcomm® Adreno™ 640
 Samsung Galaxy Note 10 (EU market) | ARM Mali G76
-Samsung Galaxy Note 10 (NA market) | Qualcomm Adreno 640
+Samsung Galaxy Note 10 (NA market) | Qualcomm® Adreno™ 640
```

Comment:
This only mentions EU and NA, should we broaden to cover the world?
I'm not sure what is the distribution of Exynos/Snapdragon for the Note10.

Reply:
fixed